guides/plugins/security.md

```diff
@@ -45,7 +45,7 @@ print(response)
 user_name = "test-user"
 user_content = {"password": "test_password", "opendistro_security_roles": []}
 
-response = client.security.put_role(user_name, body=user_content)
+response = client.security.create_user(user_name, body=user_content)
 print(response)
```

Comment:
Correct me If I am wrong. here, put_role is written by mistake . So, it should be put_user. I renamed it to create_user

Reply:
Want to add a working sample ala https://github.com/opensearch-project/opensearch-py/pull/449 to be sure?

Reply:
Yes, but in next PR :) Because, before we merge working sample PR, Security Client APIs need to be released(as of my understanding). Only then we can consume it from opensearch-py.

Reply:
You can consume `opensearch-py` from a branch and write a working sample for this so we are sure this works before releasing since there is lack of test coverage.